.github/workflows/upload_binary.yml

```diff
@@ -38,7 +42,7 @@ jobs:
       - name: Set up latest Python
         uses: actions/setup-python@v6
         with:
-          python-version: "3.12.4"
+          python-version: "3.13"
```

Comment:
I changed this from 3.12.4 -> 3.13 so that the setup-python action does not fail when installing on arm64 windows. I could also change this to a later version of 3.12 like I did in `test.yml` if is that is preferable.